components/account/AccountMoreButton.vue

```diff
@@ -21,39 +23,63 @@ const { client } = $(useMasto())
 const useStarFavoriteIcon = usePreferences('useStarFavoriteIcon')
 const { share, isSupported: isShareSupported } = useShare()
 
+function recordEngagement(dataGlean) {
```

Comment:
helper function that just calls `engagement.record` and helps us avoid code bloat